openedx/features/edly/tests/test_utils.py

```diff
@@ -499,3 +503,40 @@ def test_get_marketing_link_without_value(self):
         expected_marketing_link = ''
         marketing_link = get_marketing_link(marketing_urls, 'HONOR')
         assert expected_marketing_link == marketing_link
+
+    def test_is_course_org_same_as_site_org_for_match(self):
+        """
+        Test "is_course_org_same_as_site_org" when course org & site org match.
+        """
+        EdlySubOrganizationFactory(
+            edx_organization=OrganizationFactory(short_name='edX'),
+            lms_site=self.request.site
+        )
+        assert True == is_course_org_same_as_site_org(self.request.site, self.course.id)
+
+    def test_is_course_org_same_as_site_org_for_mismatch(self):
```

Comment:
We can merge this test case with the test above using ddt.
```python
def test_is_course_org_same_as_site_org(self, edx_org_short_name, course_id, expected_response)

    course_key = CourseKey.from_string('course_id')
    course = CourseOverviewFactory.create(id=course_key)

    assert expected_response == is_course_org_same_as_site_org(self.request.site, course.id)
```

Reply:
Done.